chromium_src/BUILD.gn

```
@@ -83,6 +83,7 @@ static_library("chrome") {
     "//components/proxy_config",
     "//components/security_state/content",
     "//content/public/browser",
+    "//electron:packed_resources",
```

I'd prefer we explicitly list the direct dependencies rather than indirectly depending on them via this target. e.g.
```suggestion
    "//services/strings",
    "//chrome/browser:dev_ui_browser_resources",
```